services/gcp/cloudkms/cryptoKeyVersions.yml

```diff
@@ -0,0 +1,77 @@
+name: Cloud KMS Crypto Key Versions
+description: >-
+  A key version contains key material used for encryption or signing.
+scope: CRITICAL
+notes: >-
+  Cloud KMS is an extremely sensitive service. Keys can be used for encryption/decryption of 
+  sensitive data or for the creation or verification of digital signatures.
+privileges:
+  create:
+    risks: [impact:spend]
+  destroy:
+    risks: [destruction:crypto, impact:encryption]
+    notes: >-
+      Destroyed key versions cannot be recovered.
+      Any data encrypted with the key version will no longer be able to be decrypted.
+  get:
+    risks: [discovery:infra]
+  list:
+    risks: [discovery:infra]
+  manageRawAesCbcKeys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CBC keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  manageRawAesCtrKeys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CTR keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  manageRawPKCS1Keys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CTR keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
+  restore:
+    risks: [impact:spend]
+    notes: >-
+      Restores a key that was scheduled for destruction.
+  update:
+    risks: [impact:dos, destruction:metadata]
+    notes: >-
+      Can be used to disable a key version. While a key version is disabled,
+      data encrypted with it cannot be accessed.
+  useToDecrypt:
+    risks: [impact:spend, exfiltration:data]
+    notes: >-
+      Can be used to decrypt data encrypted with the key version.
+  useToDecryptViaDelegation:
+    risks: [impact:spend, exfiltration:data]
+    notes: >-
+      Can be used to decrypt data encrypted with the key version through other Google Services.
+  useToEncrypt:
+    risks: [impact:spend]
+    notes: >-
+      Can be used to encrypt data with the key version.
+  useToEncryptViaDelegation:
+    risks: [impact:spend]
+    notes: >-
+      Can be used to encrypt data with the key version through other Google Services.
+  useToSign:
+    risks: [impact:spend, impact:manipulation]
```

Comment:
Question: is this also some kind of `takeover` risk as well? If an attacker can sign data it can potentially pretend to be you.

Reply:
So I don't think it fits into the takeover category, because that risk about actually gaining control over the account/domain (e.g. the attacker can actually exfiltrate your credentials), and this is different.

I understand what you're saying though and I also struggled with this-- it's an impersonation risk, and I wasn't sure where it fit. Maybe it needs its own category? I ultimately classified it as manipulation, because impersonating entities can allow you to manipulate data, but I agree it can have impacts additional to that. Maybe we want an `impact:impersonation` if we think that's a sufficiently different risk? @nbrahms thought you might have an opion here as well

Reply:
I agree that the current categories don't fit well. But I also couldn't find a good fit on the MITRE website... Maybe one of https://attack.mitre.org/tactics/TA0006/ ?

Reply:
I don’t think a signing key counts as a credential though? That page specifies account names/passwords. I also tried looking and struggled to find a fit though.

I’ll merge this and we can change it in a follow up if we come up with a more suitable risk